packages/slice-machine/server/src/api/slices/save-mock.ts

```diff
@@ -0,0 +1,55 @@
+// needs cwd, library, slice name, and the content
+
+import { ComponentMocks } from "@slicemachine/core/build/models";
+import { sliceMockPath } from "@slicemachine/core/build/node-utils";
+import { Files } from "@slicemachine/core/build/node-utils";
+import * as t from "io-ts";
+import { Response } from "express";
+import { fold } from "fp-ts/Either";
+import { pipe } from "fp-ts/lib/function";
+
+function saveSliceMockToFileSystem(
+  cwd: string,
+  libraryName: string,
+  sliceName: string,
+  mock: ComponentMocks
+) {
+  const path = sliceMockPath(cwd, libraryName, sliceName);
+  return Files.writeJson(path, mock);
+}
+
+const SaveMockBody = t.type({
```

Comment:
you can replace `t.type` by `t.strict` which combines `t.type` and `t.exact` so even if you send too many data, they are filtered on the output.
Same for `SaveMockRequest` below, even if you send the whole `env`, it would only remain the `cwd` in the output so it’s only what we need that’s even better